src/index.html

```diff
@@ -1,14 +1,80 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin ="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;1,600&family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a class="logo" href="#">
+        <img src="images/logo.png" alt="Moyo logo">
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
+
```

Comment:
Please look Dorota's mentions there is no need to add empty line space between list elements, because it is an exception, please fix formatting to be:

```
<ul>
  <li></li>
  <li></li>
  <li></li>
  <li></li>
</ul>
```